lib/deck.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Deck
+  attr_accessor :cards, :categorized_cards
+
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def count
+    @count = cards.count
+  end
+
+  def cards_in_category(category)
+    @categorized_cards = []
+    @cards.each do |card|
+      @categorized_cards << card if card.category == category
+    end
+  end
```

Comment:
We want to be able to call `cards_in_category` and return a collection of cards. The way this method is written, it is returning `@cards` (that is was `each` returns and that is the last thing your method is doing). You've written it such that we have to call _another_ method (`categorized_cards`) to actually get this list, which is not what the spec is asking for. It also adds some unnecessary code.